### DISCUSSION TITLE
Not all the metadata is showing

### DISCUSSION THREAD

**Discussion by EliasSchlie** [marcusolsson/obsidian-projects#352] [Help] [answered]
I use metadata outside the YAML area

Example:
`---`
`tags: note`
`---`
Status:: "[[Todo]]"

in a project note, only the metadata from the YAML part is shown. Is it possible to display all of it?

Thanks for the great plugin 😁

**Comment by marcusolsson** [comment, marked as answer]
Hi! 👋

Inline attributes (`::`) is a Dataview-specific feature, so you need to create a project from a Dataview query instead.

There's an open feature request for adding the inline attribute format to the frontmatter data source as well (https://github.com/marcusolsson/obsidian-projects/issues/173).

**Reply by EliasSchlie** [reply]
With dataview specific I can't change any values in the list 😢 I would love this feature to be achieved 👍
